general/index.md

```diff
@@ -113,8 +113,9 @@ When there are no other dependencies on other functions, individual functions in
 
 When performing alphabetization, ignore symbols and ignore case.  That means these elements are sorted correctly:
 
-    var $monday, Moo, moose, _mop;
+    global $monday, Moo, moose, _mop;
 
+Specific languages can override general ordering of variables, please refer to the language you are using to see if there are overriding rules.
```

Comment:
There's a disclaimer at the top of this document that indicates the same.  I don't think we should keep on saying it for every rule that we implement, so would prefer to have this line stricken.